wesnoth-ei.po

```diff
@@ -3249,7 +3281,7 @@ msgstr ""
 #. [message]: speaker=Dolburras
 #: data/campaigns/Eastern_Invasion/scenarios/06a_Undead_Crossing.cfg:334
 msgid "Glug... glug..."
-msgstr ""
+msgstr "Glo… glo..."
```

Comment:
```suggestion
msgstr "Glo... glo..."
```

Reply:
Já fakt nevím, proč mi to poslední dobou tak blbne. Tohle jsem evidentně psal na jedné klávesnici a...